mp4parse_capi/Cargo.toml

```diff
@@ -0,0 +1,34 @@
+[package]
+name = "mp4parse_capi"
+version = "0.5.0"
+authors = [
+  "Ralph Giles <giles@mozilla.com>",
+  "Matthew Gregan <kinetik@flim.org>",
+]
+
+description = "Parser for ISO base media file format (mp4)"
+documentation = "https://mp4parse-docs.surge.sh/mp4parse/"
+license = "MPL-2.0"
+
+repository = "https://github.com/mozilla/mp4parse-rust"
+
+# Cargo includes random files from the working directory
+# by default! Avoid bloating the package with test files.
+exclude = [
+  "*.mp4",
+]
+
+build = "build.rs"
+
+[dependencies]
+"mp4parse" = {path = "../mp4parse"}
```

Comment:
Oh right! https://github.com/serde-rs/serde/blob/master/serde_codegen/Cargo.toml#L36

This is not a bug and it indeed needs a version number.